src/libproc_macro/diagnostic.rs

```diff
@@ -13,7 +13,7 @@ use Span;
 use rustc_errors as rustc;
 
 /// An enum representing a diagnostic level.
-#[unstable(feature = "proc_macro", issue = "38356")]
+#[unstable(feature = "proc_macro_diagnostic", issue = "38356")]
```

Comment:
What's the point in renaming unstable library features?
This is only breaking all the code using them.